bin/buildingsync_at_modify.rb

```diff
@@ -0,0 +1,108 @@
+require 'rexml/document'
+require 'tempfile'
+
+# return line index of element
+def line_index(file, list)
+  tag = Hash.new
+
+  # find line number of <auc:Address> and </auc:Ownership>
+  File.foreach(file).with_index(1) do |line, index|
```

Comment:
this assumes that the XML has been formated "nicely"

Reply:
yes. I'm trying to convert this part to using Xpath to insert elements. But BSync files are formatted nicely from seed anyway which I think is what imma did unless I manually download 'result.xml'

Reply:
Hmm, i just picked a random file that this has to run against and it doesn't look "nicely" formatted.  File was "buildingsync_files/9473.xml". I will test locally.